go-controller/pkg/metrics/ovn_db.go

```diff
@@ -352,89 +349,75 @@ func getOvnDbVersionInfo() {
 	}
 }
 
-// ReadinessProbeCommand runs readiness probes against various targets
-var OvnDBExporterCommand = cli.Command{
-	Name:  "ovn-db-exporter",
-	Usage: "",
-	Flags: []cli.Flag{
-		&cli.StringFlag{
-			Name:  "metrics-bind-address",
-			Usage: `The IP address and port for the metrics server to serve on (default ":9476")`,
-		},
-	},
-	Action: func(ctx *cli.Context) error {
-		bindAddress := ctx.String("metrics-bind-address")
-		if bindAddress == "" {
-			bindAddress = "0.0.0.0:9476"
-		}
-
-		if err := util.SetExec(kexec.New()); err != nil {
-			return err
+func RegisterOvnDBMetrics(clientset *kubernetes.Clientset, k8sNodeName string) {
+	err := wait.PollImmediate(1*time.Second, 300*time.Second, func() (bool, error) {
+		return checkPodRunsOnGivenNode(clientset, "name=ovnkube-db", k8sNodeName, false)
```

Comment:
if this is set to false, the first time it tries, if the pod is not there we do not retry, is ok?

Reply:
Yes, that is fine. Since `ovnkube-node` POD is dependent on `ovnkube-db` POD. We have code in place in `ovnkube.sh` that waits for ovnkube-db pods to come up before it let's `ovnkube-node` pod to start. So, we are good with single successful try.